### PR TITLE
HCAP-1172: part2 Select Site modal

### PR DESCRIPTION
**Pull request by alva-fresh**
https://freshworks.atlassian.net/browse/HCAP-1172

**What's included in this PR?** 👀 
- minor improvements to `Dialog` component 
  - changes to avoid horizontal scroll-bar within a modal
- adding some logic to `SelectSite` modal for mutli-participant flow
- adding ellipsis mask for loooong site names

**What's remaining?**
- View selected participants list
- merge conflicts

https://www.figma.com/proto/35c5C0DyTEOxJ4N4DIaADL/HCAP-Design?node-id=2005%3A20369&viewport=241%2C48%2C0.36&scaling=min-zoom&starting-point-node-id=2002%3A19657

### CODE REVIEW

File: client/src/pages/private/ParticipantTableDialogues.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ export const ParticipantTableDialogues = ({
     });
   };
 
+  const handleMultiSelectProspectingSites = (values, _participantIds) => {
```
Comment (fwpushan) [inline client/src/pages/private/ParticipantTableDialogues.js:41]: Please check the status with latest master branch

**Review by fwpushan — COMMENTED** [review]
LGTM, please resolve conflicts.

**Review by taraepp — APPROVED** [review]
looks good to me

**Comment by alva-fresh** [comment]
Closing this PR; changes are captured in #694

**Pull request closed without merging**